react/rules/DEU.js

```diff
@@ -0,0 +1,60 @@
+import { isPastDate } from '../utils/dateRules'
+
+export default {
```

Comment:
Missing changelog entry. Vide:

https://github.com/vtex/profile-form/blob/master/CHANGELOG.md#added

Reply:
@salesfelipe I have updated the changelog, can you please Verify